doc/monitoring/metrics_reference.rst

```diff
@@ -708,3 +708,172 @@ Event loop tx thread information:
             -   Event loop prolog time (ms)
         *   -   ``tnt_ev_loop_epilog_time``
             -   Event loop epilog time (ms)
+
+Memtx
+-----
+
+Memtx mvcc memory statistics.
+Transaction manager consists of two parts:
+- the transactions themselves (TXN section)
+- MVCC
+
+..  _metrics-reference-memtx_txn:
+
+TXN
+~~~
+
+``statements`` are the transaction statements.
+For example, the user started a transaction and made an action in it `space:replace{0, 1}`.
+Under the hood, this operation will turn into ``statement`` for the current transaction.
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_memtx_tx_tnx_statements_total``
+            -   The number of bytes that are allocated for the statements of all transactions.
```

Comment:
Of all current transactions?

Reply:
If it is, Prometheus won't like the idea of using `_total` suffix for non-counter metric and this metric is non-counter by its nature.